include/knowhere/comp/index_param.h

```diff
@@ -48,6 +48,10 @@ constexpr const char* INDEX_DISKANN = "DISKANN";
 namespace meta {
 constexpr const char* INDEX_TYPE = "index_type";
 constexpr const char* METRIC_TYPE = "metric_type";
+constexpr const char* DATA_PATH = "data_path";
+constexpr const char* INDEX_PREFIX = "index_prefix";
+constexpr const char* VERSION_CODE = "version_code";
+constexpr const char* RETRIEVE_FRIENDLY = "retrieve_friendly";
```

Comment:
I didn't see anywhere to use these

Reply:
Milvus will use these const strings as keys.